src/libraries/System.Text.RegularExpressions/src/System/Text/RegularExpressions/RegexRunner.cs

```diff
@@ -207,6 +173,86 @@ public abstract class RegexRunner
             }
         }
 
+        internal void InitializeForScan(Regex regex, ReadOnlySpan<char> text, int textstart, bool quick)
+        {
+            this.quick = quick;
```

Comment:
I'm not clear how this argument differs from the ones mentioned below about being stored into fields?

Reply:
it doesn't, I can move it down and remove the `this.` prefix to make it consistent.